app/views/settings/profiles/show.html.erb

```diff
@@ -39,9 +39,9 @@
           </div>
           <div class="flex gap-2 items-center bg-white p-4 border border-alpha-black-25 rounded-lg">
             <div class="mr-1 flex justify-center items-center bg-gray-50 w-8 h-8 rounded-full border border-alpha-black-25">
-              <p class="uppercase text-xs text-gray-500"><%= Current.user.first_name.first %></p>
+              <p class="uppercase text-xs text-gray-500"><%= Current.user.initial %></p>
             </div>
-            <p class="text-gray-900 font-medium text-sm"><%= Current.user.first_name %> <%= Current.user.last_name %></p>
```

Comment:
Before was like this - 

![CleanShot 2024-04-26 at 14 48 01@2x](https://github.com/maybe-finance/maybe/assets/399663/18f0b1a1-a682-4e70-893a-ed14a4ab35d1)